src/App.tsx

```diff
@@ -62,9 +62,7 @@ export interface HealthCheck {
     error: string;
   };
 }
-const devLocation = process.env.NODE_ENV === 'development' || window.location.origin === 'http://51.250.66.202';
-
-export const apiUrl = devLocation ? process.env.REACT_APP_API_DEV_ADDRESS : process.env.REACT_APP_API_ADDRESS;
+export const apiUrl = process.env.REACT_APP_API_URL ? process.env.REACT_APP_API_URL : `${window.location.origin}/api/v1`;
```

Comment:
И если к локейшену ты добавляешь `/api/v1`, значит, ожидаешь в .env адрес вместе с этой строкой. Это не очень хорошо, так как я бы туда просто вбивал адрес сервера. Давай этот суфикс добавлять и тут и там.

+ REACT_APP_API_URL замени на APP_API_URL. Можно написать полностью апликейшен, чтобы было красивее, но в любом случае, без реакта должно быть.

Reply:
Вопрос, в документации прописано, что нельзя избавиться от префикса REACT_APP_
Как это необходимо реализовать тогда?

- You must create custom environment variables beginning with REACT_APP_. Any other variables except NODE_ENV will be ignored to avoid accidentally [exposing a private key on the machine that could have the same name](https://github.com/facebook/create-react-app/issues/865#issuecomment-252199527). Changing any environment variables will require you to restart the development server if it is running.